atak-civ/plugin-examples/plugintemplate/app/src/main/res/layout/create_timer_layout.xml

```diff
@@ -21,29 +21,29 @@
         />
     <LinearLayout
         android:id="@+id/nameZone"
-        android:layout_width="wrap_content"
+        android:layout_width="match_parent"
```

Comment:
I feel like the name not being aligned with the other fields is a bit strange, along with just how its much larger. I feel like Name doesn't really have a special importance, especially over something like duration so it seems strange that it has less margins and larger font

Reply:
Yeah, I was on the fence about it either way and couldn't decide, so I'll go with your suggestion.